ci/post/changelog.py

```diff
@@ -76,10 +84,13 @@ def main():
 
     changelog = {}
     url = "https://api.github.com/repos/scp-fs2open/fs2open.github.com/pulls"
+    page = 1
     last_page = MAX_PAGES
-    for page in range(1, last_page + 1):
+    while page < last_page + 1:
```

Comment:
I think it would be clearer like this:
```suggestion
    while page <= last_page:
```